LayoutTests/imported/w3c/web-platform-tests/css/css-text/text-autospace/text-autospace-combining-marks.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<link rel="help" href="https://drafts.csswg.org/css-text-4/#text-autospace-property">
+<link rel="author" title="Vitor Roriz" href="https://github.com/vitorroriz">
+<link rel="match" href="text-autospace-combining-marks-ref.html">
+<style>
+.test {
+  font-size: 40px;
+  border: solid 3px salmon;
+  width: fit-content;
+}
+.no-autospace {
+  text-autospace: no-autospace;
+}
+.normal {
+  text-autospace: normal;
+}
+</style>
+<div id="container">
+  <div class="test normal">国&#x301;国&#x301;XX&#x301;国</div>
```

Comment:
This is good, but you want to make sure you have all ordered combinations of { regular ideographic, combined ideographic, regular ASCII, combined ASCII }; you're missing a few.